build.gradle

```diff
@@ -69,6 +69,7 @@ dependencies {
     compile "mrtjp:MrTJPCore:${config.mc.version}-${config.mrtjp.version}:dev"
     compile "codechicken:CodeChickenLib:${config.mc.version}-${config.ccl.version}:dev"
     compile "codechicken:ForgeMultipart:${config.mc.version}-${config.fmp.version}:dev"
+    compile "codechicken:CodeChickenCore:${config.mc.version}-${config.ccc.version}:dev"
```

Comment:
I get this Error cause CCC 1.0.2.9 is used and this has fixed it. or what am I doing wrong?
![unbenannt](https://cloud.githubusercontent.com/assets/6798544/5950571/69986428-a75c-11e4-91f2-883bbda6041b.PNG)